azure/v25.0.0/release.yaml

```diff
@@ -109,7 +109,7 @@ spec:
   components:
   - name: cluster-azure
     catalog: cluster
-    version: 0.17.0
+    version: 0.18.0
```

Comment:
Why change this for v25.0.0? We should not modify existing releases.

Reply:
Changed the wrong file 🤦🏻